tests/nightly/test_large_array.py

```diff
@@ -1168,6 +1169,83 @@ def check_correctness(mxnet_op, numpy_op, atol=1e-3):
     check_binary_broadcast()
 
 
+def test_linalg():
+    def get_large_identity_mat():
+        A = nd.zeros((LARGE_SQ_X, LARGE_SQ_X))
+        for i in range(LARGE_SQ_X):
+            A[i,i] = 1
+        return A
+
+    def batchify(A):
+        A_np = A.asnumpy()
+        B = nd.array([A_np, A_np])
```

Comment:
instead of duplicating the matrix maybe you can just add an additional dimension by [A_np]. In my test I originally did [matrix1, matrix2] but this effectively doubled the runtime so in the end I used just [matrix] with shape (1,8000,8000)

Reply:
@Zha0q1 good idea. Does adding an additional dimension effectively calls all the batch functions we wish to test i.e., there is no difference between the functions called for (1,70k,70k) vs (2,70k,70k) ?

Reply:
I would think so. But to say 100% we would probably need to use GDB to see which function it steps into. In my case tho there is only operator for batch_syrk, not single-matrix syrk

Reply:
yeah, I checked the code. It just loops over the first dimension, so should be the same

Reply:
@mseth10  and @Zha0q1  atleast run it for 2 batches. (2,70k,70k) would be better

Reply:
(1,70k,70k) should invoke the batch version of the operator just (2,70k,70k), and essentially the batch version is just iterating the first dimension and calling the single-matrix version of the operator. The latter will take 2x time which is in my case > 40 mins. Given the number of operators that can be a huge waste